app/Http/Controllers/FavoriteController.php

```diff
@@ -40,6 +41,7 @@ public function add_favorite(Request $request){
         $favorite->urlToImage = $request->urlToImage;
         $favorite->category = $request->category;
         $favorite->country = $request->country;
+        $favorite->posted_status = 1; //default state (not_posted)
```

Comment:
This is incorrect
Make a separate call that retrieves the state by a string/name, then assign the Id if it's found